src/lib.rs

```diff
@@ -421,10 +429,8 @@ impl Conntrack {
 }
 
 unsafe fn parse_ct_attr(attr: *const nlattr, ct: &mut Conntrack) {
-    let typ = nla::get_type(attr) as c_uint;
-    match typ {
-        CTA_ID => ct.id = nla::get_u32(attr),
-        _ => (),
+    if nla::get_type(attr) as u32 == CTA_ID {
```

Comment:
I use match here because there are many more `CTA_*` constants that could be parsed, it's just that I currently only need the ID so I didn't include the rest. So I don't think changing it to `if` is good.